### PR TITLE
Add blog post for Tenzir v4.5

### PR DESCRIPTION
**Pull request by mavam**
This PR adds the blog post for the upcoming release, fixing a few docs issues along the way.

### CODE REVIEW

File: web/blog/tenzir-v4.5/index.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+- When you deploy a demo node at [app.tenzir.com](https://app.tenzir.com), it
+  now starts up faster, the pre-loaded pipelines come with labels, and have been
+  ported to use the native connector API instead of relying on cURL.
+
+- It is now possible to reference nested records in many operators that wrangle
+  data, such as `select`, `extend`, `put`, and `replace`.
+
+- The `summarize` operator now behaves more like other data frame processing
+  engine and TODO.
```
Comment (mavam) [inline web/blog/tenzir-v4.5/index.md:82]:
```suggestion
  engine and TODO.
```

@jachris mind describing the change here?

File: web/blog/tenzir-v4.5/index.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+## The API as Operator
+
+We exposed the API as a new operator called
+[`api`](/next/operators/sources/api). The benefit primarily materializes for
+developers who can now rapidly prototype integrations by using the app or
+`tenzir` command line tool, without having to do spin up the integrated web
+server and do gymnastics with `curl` and `jq`.
+
+TODO: provide an example
```
Comment (mavam) [inline web/blog/tenzir-v4.5/index.md:54]: @dominiklohmann do you have a cool example?

File: web/blog/tenzir-v4.5/index.md (hunk 1 above)
Comment (jachris) [reply web/blog/tenzir-v4.5/index.md:82]: Something like this:

- The `summarize` operator without now yields a result even if it receives no input (assuming there is no grouping with `by`). For example, `summarize num=count(foo)` returns `{"num": 0}`.

Comment (dominiklohmann) [reply web/blog/tenzir-v4.5/index.md:82]: I took this exactly, thanks Jannis!

File: web/blog/tenzir-v4.5/index.md (hunk 2 above)
Comment (dominiklohmann) [reply web/blog/tenzir-v4.5/index.md:54]: I added two.

**Review by jachris — APPROVED** [review]

**Pull request merged**